src/web/js/trove/chart-lib.js

```diff
@@ -804,11 +1190,16 @@ ${labelRow}`;
         // must append the overlay _after_ drawing to make the overlay appear
         // correctly
         root.append(overlay);
+        // return true;
       }
 
       return RUNTIME.pauseStack(restarter => {
         google.charts.setOnLoadCallback(() => {
-          setup(restarter);
+          try{
+            setup(restarter)
+          } catch (e) {
+            return restarter.error(e);
```

Comment:
Thank you! It's so frustrating to debug when we miss errors like these.

Reply:
No problem! It's all thanks to ben lerner's help!